Snowflake.Data/Core/Session/EasyLoggingStarter.cs

```diff
@@ -42,6 +54,15 @@ internal EasyLoggingStarter()
 
         public virtual void Init(string configFilePathFromConnectionString)
         {
+            if (string.IsNullOrEmpty(configFilePathFromConnectionString))
```

Comment:
maybe let's move this comments after `AllowedToInitialize()` check because otherwise we would log it for every connection.

Reply:
Done